pytest/common_lib/shared/__init__.py

```diff
@@ -239,16 +240,18 @@ def start_cluster_with_mpc(
     contract,
     presignatures_to_buffer=None,
     start_mpc_nodes=True,
+    threshold=None,
```

Comment:
So, did the tests not pass before this change?

Reply:
That takes place in `init_cluster`, see also this comment: https://github.com/near/mpc/pull/682#discussion_r2222356401